### PR TITLE
refactoring proxy and wrong boot order tests to use cluster

### PR DESCRIPTION
**Pull request by nshidlin**

### CODE REVIEW

File: discovery-infra/tests/test_wrong_boot_order.py [hunk 1]
```diff
@@ -7,35 +7,25 @@ class TestWrongBootOrder(BaseTest):
     @pytest.mark.regression
     def test_wrong_boot_order_one_node(self, api_client, nodes, cluster):
```
Comment (lalon4) [inline discovery-infra/tests/test_wrong_boot_order.py:8]: please remove api_client

Comment (nshidlin) [reply discovery-infra/tests/test_wrong_boot_order.py]: changed

**Review by YuviGold — APPROVED** [review]
LGTM

**Pull request merged**